빌드 에러 해결

## 🐛 어떤 버그인가요?

> 어떤 버그인지 간결하게 설명해주세요



### 버그 재현 방법
311번 PR 이후 pull push 시 빌드 에러 발생
### 스크린샷 첨부
```
git push
Enter passphrase for key '/Users/haheungju/.ssh/id_ed25519':

> oz_externship_fe_template@0.0.0 lint:strict
> eslint . --config eslint.config.strict.js


> oz_externship_fe_template@0.0.0 build
> tsc -b && vite build

src/components/chat/ChatList.tsx:108:22 - error TS2304: Cannot find name 'useRef'.

108   const LoadingRef = useRef<HTMLDivElement | null>(null)
                         ~~~~~~


Found 1 error.

husky - pre-push script failed (code 2)
error: failed to push some refs to 'github.com:ThePott/forked_oz_externship_fe_03_team3.git'
❯
```
## 🛠️ 수정 방향

> 어떻게 수정해야 하나요?
임포트 추가

- [ ] (수정 사항을 적어주세요)
- [ ]

--- a/src/components/chat/ChatList.tsx
+++ b/src/components/chat/ChatList.tsx
@@ -1,7 +1,7 @@
 import ChattingLayout from '@/components/layout/chattingRoom/ChattingLayout'
 import ChatListCard from './feat/ChatListCard'
 import useStudyHubStore from '@/store/store'
-import { useEffect } from 'react'
+import { useEffect, useRef } from 'react'
 import ChatListSkeleton from './skeleton/ChatListSkeleton'
 import useOneWayInfinityScroll from '@/hooks/useOneWayInfinityScroll'
 import ChatListSkeletonCard from './skeleton/ChatListSkeletonCard'

